src/main/java/core/basesyntax/model/Truck.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax.model;
+
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started.");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped.");
```

Comment:
Similarly, the message in the `stopWork` method should be more informative. It should indicate that the type of `Machine` that has stopped working is `Truck`. Consider rephrasing it to something like 'Truck has stopped working.'